### ISSUE TITLE
Create ECR repo

### ISSUE THREAD

**Issue by chriscasto** [metrotranscom/doorway-infra#13]
Create an ECR repo for our Docker images

**Comment by isaac-looker** [comment]
Is the plan to use a single repo for all images or have we considered a dev and prod repo? I'm not familiar with IAM to ECR permission boundries but it may be helpful to create a boundry at the resource(ECR) level.

**Comment by chriscasto** [comment]
TL;DR - I think two repos, one for prod and one for pre-prod, would be good.

I strongly recommend separating prod and non-prod resources as much as possible.  In an ideal world, the prod environment would be in a completely separate account that is fully isolated from anything non-prod at every level.  Those boundaries become less meaningful between non-prod environments (dev, test, qa, staging, etc), but an org with a dedicated QA or UAT teams may even want to prevent the dev team from mutating those environments outside of the proper pipelines.  This is an area where we have to compromise between flexibility and modularity (since this is an open source project that will be used by different organizations with wildly different needs, norms, and capabilities) and "good enough" so that it doesn't become a blocker.  I think using two environments would keep us from adopting shortcuts and optimizations that would be difficult to roll back if things needed to change in the future.
